FirebaseRemoteConfig/Tests/Unit/RCNConfigContentTest.m

```diff
@@ -262,4 +291,38 @@ - (void)testCopyFromDictionaryUpdatesActiveConfig {
                         [_configContent.activeConfig[@"dummy_namespace"][@"new_key"] dataValue]);
 }
 
+- (void)testCheckAndWaitForInitialDatabaseLoad {
+  RCNConfigDBManagerMock *mockDBManager = [[RCNConfigDBManagerMock alloc] init];
+  RCNConfigContent *configContent = [[RCNConfigContent alloc] initWithDBManager:mockDBManager];
+
+  // Check that no one of first three calls of `-checkAndWaitForInitialDatabaseLoad` do not produce
+  // timeout error <begin>
+  XCTestExpectation *expectation1 =
+      [self expectationWithDescription:
+                @"1st `checkAndWaitForInitialDatabaseLoad` return without timeout"];
+  dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+    [configContent checkAndWaitForInitialDatabaseLoad];
```

Comment:
should we assert that `checkAndWaitForInitialDatabaseLoad` returns true here too?

Reply:
It would be cool to have this assert while it not strictly necessary. I will add it now.

Additional info: it seems safe to put XCTAssert in non-main queue. 
Apple example: https://developer.apple.com/documentation/xctest/asynchronous_tests_and_expectations/testing_asynchronous_operations_with_expectations?language=objc
Callback in example "is executed on the delegate queue" https://developer.apple.com/documentation/foundation/nsurlsession/1410330-datataskwithurl?language=objc

Reply:
@karenyz ready :)